build.Dockerfile

```
@@ -55,14 +58,20 @@ ENV ADDITIONAL_FEATURES=$ADDITIONAL_FEATURES_ARG
 
 ARG FINGERPRINT=none
 
+ARG SGX_COMMERCIAL_KEY=enclave-runtime/Enclave_private.pem
+ENV SGX_COMMERCIAL_KEY ${SGX_COMMERCIAL_KEY}
+
+ARG SGX_PASSFILE
+ENV SGX_PASSFILE ${SGX_PASSFILE}
+
 WORKDIR $WORKHOME/worker
 
 COPY . .
 
 RUN --mount=type=cache,id=cargo-registry,target=/opt/rust/registry \
 	--mount=type=cache,id=cargo-git,target=/opt/rust/git/db \
 	--mount=type=cache,id=cargo-sccache-${WORKER_MODE}${ADDITIONAL_FEATURES},target=/home/ubuntu/.cache/sccache \
-	echo ${FINGERPRINT} && make && cargo test --release && sccache --show-stats
+	echo ${FINGERPRINT} && make && make mrenclave && make mrsigner && cargo test --release && sccache --show-stats
```

I think it would be help with readability if we used the `identity` target, instead of it's separate components.
```suggestion
	echo ${FINGERPRINT} && make && make identity && cargo test --release && sccache --show-stats
```